tests/testDipoleFitter.py

```diff
@@ -56,86 +55,74 @@ class DipoleFitTestGlobalParams(object):
     @var offsets: pixel coordinates between lobes of dipoles
     """
 
-    def __init__(self):
+    def __init__(self, xc=np.array([65.3, 24.2]), yc=np.array([38.6, 78.5])):
         """Initialize the parameters."""
```

Comment:
The problem is if the default thingies are mutable. If at some point the class decides that `xc` or `yc` should be modified then there is a good change that things will get confused. The safe option is to say `xc=None` and then check for `xc is None` in the function itself. Given this is test code you might get away with it.